scripts/validate.py

```diff
@@ -51,6 +52,11 @@ def run(global_config):
             "Expecting non-empty IGNORED_FILES. Please check if ignored_files_manifest.csv is configured correctly!!!"
         )
 
+    logger.info(
+        "If validation job is run with FORCE_CREATE_MANIFEST True: errors from missing objects are to be expected due to redaction of records in the data release following the current run"
+    )
+    FORCE_CREATE_MANIFEST = global_config.get("FORCE_CREATE_MANIFEST")
```

Comment:
Can you do `get("FORCE_CREATE_MANIFEST", False)` we need a default value if value is not provided